src/rest/test/test_translation.py

```diff
@@ -0,0 +1,3 @@
+# TODO
+
+# TODO Translation not realy testable, maybe the negatives yes
```

Comment:
I think testing this with "contains" is possible. German input "Der Hund ist braun". Translation should contain "dog" and "brown". This would at least at a base level of test safety